packages/validation/src/loadbalancers.schema.ts

```diff
@@ -152,6 +154,91 @@ export const UpdateRouteSchema = object({
   }),
 });
 
+// Endpoint Schema
+const endpointSchema = object({
+  ip: string().when('hostname', {
+    is: (hostname: string) => !hostname,
+    then: string().required(),
+    otherwise: string(),
+  }),
+  hostname: string().when('ip', {
+    is: (ip: string) => !ip,
+    then: string().required(),
+    otherwise: string(),
+  }),
+  port: number().integer().required(),
+  capacity: number().integer().required(),
+});
+
+// Service Target Schema
+const createLoadBalancerServiceTargetSchema = object({
```

Comment:
```suggestion
const CreateLoadBalancerServiceTargetSchema = object({
```
Nitpick: throughout the codebase, it looks like our naming for schemas is inconsistent as to whether we start with a capital letter or not... but can we stay consistent with the existing capitalization of schema objects in this file?